content/en/docs/pipelines/OWNERS

```diff
@@ -0,0 +1,6 @@
+approvers:
+  - Bobgy
+  - IronPan
+  - neuromage
```

Comment:
add Joe to approver

Reply:
joe is already root folder approver, there's no need to duplicate here